<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LTOBDemoClass</class>
 <widget class="QWidget" name="LTOBDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LTOBDemo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="topControlsLayout">
     <item>
      <widget class="QSpinBox" name="spinTargetCount">
       <property name="minimum">
        <number>10</number>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
       <property name="value">
        <number>200</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnGenerate">
       <property name="text">
        <string>生成</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCustomPlot" name="plotOriginal" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qCustomPlot/qcustomplot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="LTOBDemo.qrc"/>
 </resources>
 <connections/>
</ui>
